tests/expressions/tests.py

```diff
@@ -516,6 +516,13 @@ def test_annotations_within_subquery(self):
             [{'salary': 10, 'total_employees': 2300}, {'salary': 20, 'total_employees': 35}],
         )
 
+    def test_subquery_references_joined_table_twice(self):
+        inner = Company.objects.filter(num_chairs__gte=OuterRef('ceo__salary'),
```

Comment:
Hmm. Re: the comment - can there be more than one alias? I don't think I understand enough about what that could be.